Tests/BackendIntegrationTests/OfflineStoreKitIntegrationTests.swift

```diff
@@ -250,6 +250,7 @@ class OfflineStoreKit1IntegrationTests: BaseOfflineStoreKitIntegrationTests {
         self.serverDown()
 
         try await self.purchaseMonthlyProduct(allowOfflineEntitlements: true)
+        try await Task.sleep(for: .seconds(1))
```

Comment:
Might be worth adding some comments explaining the reason for this one?

Reply:
Yep, good point! It can be short, saying that we want to make sure that the two transactions has a different `purchase_date`. Thanks for the suggestion!